auto_submit/lib/validations/ci_successful.dart

```diff
@@ -54,14 +96,20 @@ class CiSuccessful extends Validation {
         failures.add(FailureDetail('tree status $treeStatusName', 'https://flutter-dashboard.appspot.com/#/build'));
       }
     }
-    // List of labels associated with the pull request.
-    final List<String> labelNames = (messagePullRequest.labels as List<github.IssueLabel>)
-        .map<String>((github.IssueLabel labelMap) => labelMap.name)
-        .toList();
+  }
+
+  /// Validate the ci build test run statuses to see which have succeeded and
```

Comment:
Nit: dart doc style https://dart.dev/guides/language/effective-dart/documentation#do-separate-the-first-sentence-of-a-doc-comment-into-its-own-paragraph

Reply:
Not a nit, it is good to keep consistent with requirements. I have fixed this.